src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -97,7 +97,7 @@ private static String describeType(ParameterType type, int headerLevel) throws E
             }
         } else if (type instanceof ErrorType) { //Shouldn't hit this; open a ticket
             Exception x = ((ErrorType) type).getError();
-            if(x instanceof NoStaplerConstructorException) {
+            if(x instanceof NoStaplerConstructorException || x instanceof UnsupportedOperationException) {
```

Comment:
While this actually shows up as an error message within the generated docs, I decided to display something more user friendly.  As it was mentioned before, it's not really great to show errors on a website!